src/sagemaker/session.py

```diff
@@ -6797,6 +6800,41 @@ def get_execution_role(sagemaker_session=None):
 
     if ":role/" in arn:
         return arn
+
+    if use_default:
```

Comment:
check if the default role already exists before creating. if so, then use that.

Reply:
I did that in the try except block below